R/bind.r

```diff
@@ -137,6 +137,9 @@ bind_cols <- function(...) {
   names(dots)[is_data_frame] <- ""
 
   out <- vec_cbind(!!!dots)
+  if (!any(map_lgl(dots, function(.x) is.data.frame(vec_ptype(.x))))) {
```

Comment:
Is the `vec_ptype()` necessary here? it looks like this could be:

```suggestion
  if (!any(map_lgl(dots, is.data.frame))) {
```

Reply:
oh yeah of course. Thanks.